contrib/bash_completion.d/README.md

```diff
@@ -0,0 +1,7 @@
+BASH completion script for OpenDNSSEC
+=====================================
+Copy the script to /etc/bash_completion.d/ to install. The BASH shell will
+try to autocomplete commands and options for ods-enforcer and ods-signer.
+
+The script is provided 'as is' and is not covered by our testing framework. We
+do keep it up to date and accept improvements.
```

Comment:
"As-is" and "keep it up to date" are conflicting.  I think the script is valuable and nice for you to maintain, but we do not need to make the claim to keep it up to date.  I would just remove the last paragraph or line.